Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8.3 as builder
+WORKDIR /go/src/tiedot/
+COPY ./    .
+WORKDIR /go/
+RUN go get -v -d tiedot
+RUN CGO_ENABLED=0 GOOS=linux go install -a -installsuffix cgo -v tiedot
```

Comment:
I'm not sure all of these build flags are needed. This could maybe be simplified.

I'd suggest making `CGO_ENABLED` and `GOOS` Dockerfile `ARG`s.

Reply:
Regarding CGO_ENABLED: sometimes go compiler has an odd habit of unnecessarily linking against system's libc, unless CGO_ENABLED is explicitly turned off.

tiedot does not require linking against libc, therefore it should be safe to permanently turn it off.